packages/data-point/README.md

```diff
@@ -3301,6 +3302,49 @@ function resolve(acc:Accumulator, resolveReducer:function):Promise<Accumulator>
 
 Example at: [examples/custom-entity-type.js](examples/custom-entity-type.js)
 
+## <a name="debugging">Debugging</a>
+
+When a reducer throws an error, DataPoint adds two properties to the error object:
+
+`_value:*` - the input value to the reducer that failed
```

Comment:
Re-reading this, changing this to `_input` makes more sense.

Reply:
done